src/main/java/net/imagej/ops/image/ImageNamespace.java

```diff
@@ -456,6 +457,80 @@
 		return result;
 	}
 
+	// -- quality --
+
+	@OpMethod(op = net.imagej.ops.image.quality.DefaultMAE.class)
+	public <T extends RealType<T>> DoubleType mae(final DoubleType out,
+		final IterableInterval<T> reference, final IterableInterval<T> test)
+	{
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.image.quality.DefaultMAE.class, out, reference, test);
```

Comment:
This should be `ops().run(Ops.Image.MAE.class, out, reference, test);`. That way if someone creates a more specific/specialized MAE op, it could be matched. Otherwise, `DefaultMAE` would always be used.